transports/transport-highway/src/main/java/io/servicecomb/transport/highway/HighwayClient.java

```diff
@@ -100,22 +105,26 @@ public void send(Invocation invocation, AsyncResponse asyncResp) throws Exceptio
     tcpClientPool.send(tcpClient, clientPackage, ar -> {
```

Comment:
we can add a metric item to show net time.
if too complex,  create a new issue for it

Reply:
i think this may do later in a new issue